analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -117,5 +117,19 @@ public void SimpleAssignment_ToUnsupported_FromLiteral(string snipet)
             collector.Validate("Literal: 42", x => x.State[x.Operation].HasConstraint(DummyConstraint.Dummy).Should().BeTrue("it's scaffolded"));
             collector.ValidateTag("Target", x => (x?.HasConstraint(DummyConstraint.Dummy) ?? false).Should().BeFalse());
         }
+
+        [TestMethod]
+        public void Conversion_ToLocalVariable_FromTrackedSymbol_ExplicitCast()
+        {
+            var collector = SETestContext.CreateCS(@"int a = 42; byte b = (byte)a; Tag(""b"", b);", new LiteralDummyTestCheck()).Validator;
+            collector.ValidateTag("b", x => x.HasConstraint(DummyConstraint.Dummy).Should().BeTrue());
```

Comment:
Let's assert that there exists a conversion operator in the CFG. For example by `collector.ValidateOrder`.